client/kubeconfig.go

```diff
@@ -0,0 +1,105 @@
+package client
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+)
+
+const DEFAULT_KUBECONFIG_PATH = ".kubeconfig"
+
+type KubeconfigPathOption string
+
+// WithSilent will stop the function from printing information to the console.
+const WithSilent KubeconfigPathOption = "silent"
+
+// LookupKubeconfigPath checks if a kubeconfig file exists, and returns its path.
+//
+// It first checks the "STACC_KUBECONFIG" environment variable, followed
+// by looking in the current directory.
+//
+// Use the WithSilent KubeconfigPathOption to stop this function from
+// logging helpful information directly to the console.
+func LookupKubeconfigPath(options ...KubeconfigPathOption) (string, error) {
+	var kubeconfigPath string
+
+	useSilent := false
+	for _, option := range options {
+		switch option {
+		case WithSilent:
+			useSilent = true
+		default:
+			return "", fmt.Errorf("unknown option %q", option)
+		}
+	}
+
+	var useEnvVar bool
+	staccKubeconfigEnv := os.Getenv("STACC_KUBECONFIG")
+	if staccKubeconfigEnv != "" {
+		useEnvVar = true
+		kubeconfigPath = staccKubeconfigEnv
```

Comment:
A path from `STACC_KUBECONFIG` is not explicitly converted into an absolute path here, unlike the old version. It may not actually matter, if `os.Stat()` implicitly converts into an absolute path anyway, but that's not documented 🤷 But if so, we could just remove the `filepath.Abs()` stuff completely.

Reply:
I don't like the thought of converting an environment variable used several places into an absolute path, as behaviour might change. If it breaks one place it should break everywhere.

(From what I can tell `os.Stat()` isn't converted into an absolute path)